### PR TITLE
REP-69 Update documentation

### LINKED ISSUE #69: Update documentation
**Describe the task**
Update the documentation to mention that confluence metadata cannot be replicated, and to specify what scenarios require replication to be installed on both the source and destination.

### PR DESCRIPTION
**Pull request by kcover**
#### What does this PR do?
Updates the documentation to mention that confluence metadata can't be
replicated and why replication should be installed on both the source
and destination.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@peterhuffer @clockard @paouelle @mcalcote 

#### How should this be tested? (List steps with links to updated documentation)
Look at the docs

#### What are the relevant tickets?
Fixes #69 

#### Screenshots (if appropriate)
confluence
warning

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by kcover** [comment]
The platform placeholder is currently "DDF Base" should I change it to something like "DDF Based Platform" or "DDF Based Node"? Something else?

**Comment by peterhuffer** [comment]
> The platform placeholder is currently "DDF Base" should I change it to something like "DDF Based Platform" or "DDF Based Node"? Something else?

I think DDF Based Node sounds good.

File: docs/src/main/resources/content/_using/replication-limitations.adoc [hunk 1]
```diff
@@ -31,3 +31,7 @@ result in the replication of records from the Connected Sources in addition to a
 
 Derived resources, from products such as NITFs, will not be replicated. New derived resources will,
 however, be generated by the receiving ${platform}.
+
+=== Confluence Sources
+
+Metadata received from a confluence source cannot be replicated.
```
Comment (mcalcote) [inline docs/src/main/resources/content/_using/replication-limitations.adoc]: ✏️ Should we wrap this in a note?
```
[NOTE]
====
Metadata received from a confluence source cannot be replicated.
====
```

**Comment by paouelle** [comment]
@peterhuffer What happens in the future when we support replication to Ion? DDF-based is limited.

File: docs/src/main/resources/content/_using/replication-limitations.adoc (hunk 1 above)
Comment (clockard) [reply docs/src/main/resources/content/_using/replication-limitations.adoc]: Might mention that any source that generates metacards on the fly (like Confluence) will not work with replication. 
Also should capitalize Confluence

Comment (kcover) [reply docs/src/main/resources/content/_using/replication-limitations.adoc]: I know that it's a small addition but is there any other reason I should make it a note? I think this is a good place for it but it seems odd to have a section with just a note and no other writing.

@clockard Could you explain in more detail how confluence generates metacards and why we can't replicate them? That should help me make the documentation more clear.

**Comment by peterhuffer** [comment]
Reviewed html docs. Successful hero ✅

**Pull request merged**